.eslintrc

```diff
@@ -1,11 +1,30 @@
 {
 	"extends": [
-		"xo",
-		"xo-typescript",
-		"xo-react",
+		"eslint:recommended",
+		"plugin:react/recommended",
+		"plugin:@typescript-eslint/recommended",
+		"plugin:prettier/recommended"
 	],
+	"plugins": [
+		"react",
+		"@typescript-eslint",
+		"prettier"
+	],
+	"env": {
+		"browser": true,
+		"jasmine": true,
+		"jest": true
+	},
+	"settings": {
+		"react": {
+			"pragma": "React",
+			"version": "detect"
+		}
+	},
+	"parser": "@typescript-eslint/parser",
 	"rules": {
-		"react-hooks-nesting": "error",
-		"jsx-boolean-value": false
+		"@typescript-eslint/ban-ts-comment": "warn",
```

Comment:
I assume we can get rid of this if we fix whatever needs the `@ts-ignore` comments?

Reply:
true, but don't know how to get rid of the ts issues 😅

Reply:
🤷